dota_match_scheduler/dota_match_scheduler/models.py

```diff
@@ -0,0 +1,73 @@
+import enum
+from sqlalchemy import (
+    Column,
+    Integer,
+    String,
+    DateTime,
+    BigInteger,
+    Enum,
+    create_engine,
+    ForeignKey,
+    Text,
+)
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+from variables import DB
+
+
+Base = declarative_base()
+
+
+class MyEnum(enum.Enum):
+    one = "Bo1"
+    two = "Bo2"
+    three = "Bo3"
+    four = "Bo5"
```

Comment:
Just questioning this line of code, is it meant to be Bo4 or is Bo5 okay?

Reply:
Bo4 means you can have a draw, Bo5 is usually reserved for GF matches

Reply:
Amazing! All looks good to me then :)